app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -131,16 +133,23 @@ end
 -%>
 
 rootpw --iscrypted <%= root_pass %>
+<%# Firewall %>
```

Comment:
I still see many empty lines in the rendered output and I think you can leave them out
```suggestion
<%# Firewall -%>
```